target/linux/pistachio/base-files/etc/init.d/bt-config

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh /etc/rc.common
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+START=85
+STOP=85
+
+config_bluetooth() {
+	. /lib/functions/mac.sh
+	. /lib/functions/bluetooth.sh
+
+	local IS_BT_RUNNING
+	local timeout=10
```

Comment:
10 seconds limit? How long does is usually take as this going to add a lot of time to boot